src/modV.js

```diff
@@ -1,53 +1,59 @@
 const EventEmitter2 = require('eventemitter2').EventEmitter2;
 const shaderInit = require('./shader-env');
 const threeInit = require('./three-env');
+const Layer = require('./Layer');
 const MM = require('./media-manager');
 require('./fragments/array-contains');
 require('script-loader!../libraries/beatdetektor.js');
 
+
 /**
  * modV
  * @extends {EventEmitter2}
  */
-class modV extends EventEmitter2 {
+class ModV extends EventEmitter2 {
```

Comment:
(for history) I think it's fine, naming the class with an uppercase symbol is perfectly fine, it does differentiate between the class and instance which can be useful.
Let's make sure in the documentation we name `ModV` as `modV` however as it is presented to the user as `modV`